install/windows/pwsafe.nsi

```diff
@@ -563,37 +613,61 @@ Section "Uninstall"
   Delete "$INSTDIR\pws_at.dll"
   Delete "$INSTDIR\pws_osk.dll"
   Delete "$INSTDIR\pwsafe.chm"
+  Delete "$INSTDIR\pwsafe.chw"
   Delete "$INSTDIR\pwsafe.xsd"
   Delete "$INSTDIR\pwsafe.xsl"
   Delete "$INSTDIR\pwsafe_filter.xsd"
   Delete "$INSTDIR\KPV1_to_PWS.xslt"
   Delete "$INSTDIR\KPV2_to_PWS.xslt"
   Delete "$INSTDIR\LICENSE"
-  Delete "$INSTDIR\README.TXT"
+  Delete "$INSTDIR\README.md"
   Delete "$INSTDIR\ReleaseNotes.txt"
   Delete "$INSTDIR\ReleaseNotes.html"
   Delete "$INSTDIR\ChangeLog.txt"
+  Delete "$INSTDIR\pwsafeCZ.chm"
+  Delete "$INSTDIR\pwsafeCZ.chw"
+  Delete "$INSTDIR\pwsafeCZ.dll"
   Delete "$INSTDIR\pwsafeDA.chm"
+  Delete "$INSTDIR\pwsafeDA.chw"
   Delete "$INSTDIR\pwsafeDA.dll"
   Delete "$INSTDIR\pwsafeDE.chm"
+  Delete "$INSTDIR\pwsafeDE.chw"
   Delete "$INSTDIR\pwsafeDE.dll"
   Delete "$INSTDIR\pwsafeES.chm"
+  Delete "$INSTDIR\pwsafeES.chw"
   Delete "$INSTDIR\pwsafeES.dll"
   Delete "$INSTDIR\pwsafeFR.chm"
+  Delete "$INSTDIR\pwsafeFR.chw"
   Delete "$INSTDIR\pwsafeFR.dll"
+  Delete "$INSTDIR\pwsafeHU.chm"
+  Delete "$INSTDIR\pwsafeHU.chw"
+  Delete "$INSTDIR\pwsafeHU.dll"
   Delete "$INSTDIR\pwsafeIT.chm"
+  Delete "$INSTDIR\pwsafeIT.chw"
   Delete "$INSTDIR\pwsafeIT.dll"
   Delete "$INSTDIR\pwsafeKO.chm"
+  Delete "$INSTDIR\pwsafeKO.chw"
   Delete "$INSTDIR\pwsafeKO.dll"
   Delete "$INSTDIR\pwsafeNL.chm"
+  Delete "$INSTDIR\pwsafeNL.chw"
   Delete "$INSTDIR\pwsafeNL.dll"
   Delete "$INSTDIR\pwsafePL.chm"
+  Delete "$INSTDIR\pwsafePL.chw"
   Delete "$INSTDIR\pwsafePL.dll"
   Delete "$INSTDIR\pwsafeRU.chm"
+  Delete "$INSTDIR\pwsafeRU.chw"
   Delete "$INSTDIR\pwsafeRU.dll"
+  Delete "$INSTDIR\pwsafeSL.chm"
+  Delete "$INSTDIR\pwsafeSL.chw"
+  Delete "$INSTDIR\pwsafeSL.dll"
   Delete "$INSTDIR\pwsafeSV.chm"
+  Delete "$INSTDIR\pwsafeSV.chw"
   Delete "$INSTDIR\pwsafeSV.dll"
+  Delete "$INSTDIR\pwsafeTR.chm"
+  Delete "$INSTDIR\pwsafeTR.chw"
+  Delete "$INSTDIR\pwsafeTR.dll"
   Delete "$INSTDIR\pwsafeZH.chm"
+  Delete "$INSTDIR\pwsafeZH.chw"
```

Comment:
Are globs supported here? Is it possible to write this as `$INSTDIR\pwsafe*.chw` & `$INSTDIR\pwsafe*.dll`?

Reply:
Yes, it seems that globs are supported:  http://nsis.sourceforge.net/Docs/Chapter4.html#delete. But I thought that we try to delete only files that were (possibly) created by our installer or after using pwsafe or "help" (chw are created after switching to "Index" tab).